build-scripts/rcm-guest/push-to-mirrors.sh

```diff
@@ -73,7 +74,7 @@ else
 fi
 
 # dereference the symlink to the actual directory basename: e.g. "2017-06-09.4"
-LASTDIR=$(readlink ${PUDDLEDIR})
+LASTDIR=$(readlink ${PUDDLEDIR})_${FULL_VERSION}
```

Comment:
Here (and similarly in `push-openshift-online-to-mirrors.sh`) I'm assuming we
want to keep the puddle directory in `rcm-guest` unchanged (i.e. without the
build version).

Reply:
I would prefer we change it everywhere.

Reply:
Can we also prefix the version with a 'v'? date_vVERSION.